.changeset/pre.json

```diff
@@ -0,0 +1,17 @@
+{
+  "mode": "pre",
+  "tag": "next",
```

Comment:
going to try exiting pre on this branch and fixing up changesets

🧠 https://github.com/changesets/changesets/blob/main/docs/prereleases.md

Reply:
kk seems good now, bumped all to 0.5.0!